apis/Google.Cloud.Spanner.Data/docs/index.md

```diff
@@ -254,3 +254,34 @@ If you want to diagnose session management issues,
 the statistics for the session pool associated with the connection:
 
 {{sample:SpannerConnection.GetSessionPoolSegmentStatistics}}
+
+## RpcException from SpannerException
```

Comment:
I think this heading needs to have a verb, e.g. "Obtaining an RpcException from a SpannerException" (or possibly Retrieving instead of Obtaining).